src/feature/Client/components/Filters/Filter.scss

```
@@ -0,0 +1,49 @@
+@import "assets/styles/variables.scss";
+.active{
+    color: $blue;
+}
+
+.categoryItem{
+    border: 1px solid $gray;
+    padding: .5rem;
+
+    &:hover{
+        color: $blue;
+    }
+}
+
+.categoryTitle{
```

Đặt tên class là snake thay vì camelCase nhé
```suggestion
.category-title{
```